### PR TITLE
WIP DB-732 Asterisk Logging script

### PR DESCRIPTION
**Pull request by sincejune**
Example output:

```
Reading file: logger.conf
Reading file: test.conf
Reading file: logger_general_custom.conf
File logger_general_custom.conf is not found
Your pattern is: '\[%{GREEDYDATA:timestamp:datetime:yyyy-MM-dd HH:mm:ss}\] %{WORD:level:meta}\[%{INT:lwp:int}\] %{JAVAFILE:source:meta}: %{GREEDYDATA:message}'
```

### CODE REVIEW

File: loggings/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+### Asterisk Logging Script
```
Comment (natalie-arsky) [inline loggings/README.md]: Listing limitations are fine.

File: loggings/asterisk/asterisk.py [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+            if not line.startswith("#include"):
+                buffer += line
+            else:
+                _, file = line.rsplit(maxsplit=2)
+                include_filename = os.path.join(os.path.dirname(filename), file)
+                buffer += read_config_file(include_filename)
+
+
+if __name__ == '__main__':
+    if locale.getdefaultlocale()[0] != 'en_US':
```
Comment (natalie-arsky) [inline loggings/asterisk/asterisk.py]: I am not sure how locale is used. Locale is checked and locale is imported but I don't see it used...

File: loggings/asterisk/asterisk.py [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+        print("This machine is not in the locale of 'en_US'. This may break the dashbase parsing because Asterisk "
+              "will output log according to the current locale")
+
+    # TODO input config file path
```
Comment (natalie-arsky) [inline loggings/asterisk/asterisk.py]: Is logger.conf is an example of asterisk logger configuration file which includes test.conf? Or logger.conf is configuration for asterisk.py?

File: loggings/logger.conf [hunk 4]
```diff
@@ -0,0 +1,4 @@
+[general]
```
Comment (natalie-arsky) [inline loggings/logger.conf]: Both logger.conf and test.conf are examples of configuration. Can we move them to tests folder?

File: unittest.sh [hunk 5]
```diff
@@ -0,0 +1 @@
+python -m unittest loggings/tests/asterisk.py
```
Comment (natalie-arsky) [inline unittest.sh]: Is it possible to move unittest.sh into loggings (that will be renamed) instead of having it at the root of this repo?

**Comment by sincejune** [comment]
Closing this due to related code is already merged.

**Pull request closed without merging**